jujugui/static/gui/src/app/components/basic-table/basic-table.js

```diff
@@ -76,12 +76,13 @@ class BasicTable extends React.Component {
   */
   _generateContent() {
     let rows = this.props.rows;
+    if (this.props.filter) {
```

Comment:
This is really a filterPredicate rather than a filter func. It takes a single row and must return a boolean. I'd change the name.